spec/integration/missing_relationships_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+describe "Collections with missing files" do
+
+  let(:collection)    { "x346dj04v" }
+  let(:files)         { ["x346dj06d", "x346dj08z"] }
+  let(:missing_file)  { "x346dj07p" }
+
+  before do
+    FedoraMigrate::ObjectMover.new(FedoraMigrate.find("scholarsphere:"+collection), ExampleModel::Collection.new(collection)).migrate
```

Comment:
Minor general point, but I'd find it a little more readable to use variable interpolation within the string rather than use the concatenation operator.